src/cdn/polyfills/spa.js

```diff
@@ -4,4 +4,27 @@
  */
 
 import '../polyfills.js'
-import '../spa'
+import { Agent } from '../loaders/agent'
+
+import { Instrument as InstrumentPageViewEvent } from '../../features/page_view_event/instrument'
+import { Instrument as InstrumentPageViewTiming } from '../../features/page_view_timing/instrument'
+import { Instrument as InstrumentMetrics } from '../../features/metrics/instrument'
+import { Instrument as InstrumentErrors } from '../../features/jserrors/instrument'
+import { Instrument as InstrumentXhr } from '../../features/ajax/instrument'
+import { Instrument as InstrumentSessionTrace } from '../../features/session_trace/instrument'
+import { Instrument as InstrumentSpa } from '../../features/spa/instrument'
+import { Instrument as InstrumentPageAction } from '../../features/page_action/instrument'
+
+new Agent({
+  features: [
+    InstrumentXhr,
+    InstrumentPageViewEvent,
+    InstrumentPageViewTiming,
+    InstrumentSessionTrace,
+    InstrumentMetrics,
+    InstrumentPageAction,
+    InstrumentErrors,
+    InstrumentSpa
+  ],
+  loaderType: 'spa'
```

Comment:
Could even change this loaderType now to indicate its the polyfill build for SM purposes

Reply:
@patrickhousley https://source.datanerd.us/agents/angler/pull/490